pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
+
+- Download file using remote name:
+
+`curl -O {{http://example.com/filename}}`
+
+- Automatically continue/resume previous file transfer:
```

Comment:
That seems like an auxiliary function which IMHO should be the default behavior. Perhaps we could include it in the first example (`-o`), mentioning what it does, and maybe even include it in all the other examples where it makes sense.